docs/source/install/monitoring_stack.rst

```diff
@@ -10,12 +10,12 @@ For evaluation, you can run Scylla Monitoring Stack on any server (or laptop) th
 
 .. include:: min-prod-hw.rst
 
-.. _`Scylla Prometheus API`: monitoring_apis#prometheus
+.. _`Scylla Prometheus API`: /reference/monitoring_apis#prometheus
 
 Prerequisites
 -------------
 
-* Follow the Installation Guide and install `docker`_ on the Scylla Monitoring Stack Server. This server can be the same server that is running Scylla Manager. Alternatively, you can `Deploy Scylla Monitoring Stack Without Docker <monitor_without_docker>`_ .
+* Follow the Installation Guide and install `docker`_ on the Scylla Monitoring Stack Server. This server can be the same server that is running Scylla Manager. Alternatively, you can `Deploy Scylla Monitoring Stack Without Docker </install/monitor_without_docker>`_ .
```

Comment:
Internal links are constructed with the ":doc:" directive. For more information, see [Links](https://sphinx-theme.scylladb.com/stable/examples/links.html).

```suggestion
* Follow the Installation Guide and install `docker`_ on the Scylla Monitoring Stack Server. This server can be the same server that is running Scylla Manager. Alternatively, you can :doc:`Deploy Scylla Monitoring Stack Without Docker <monitor_without_docker>` .
```